examples/kaggle/kaggle_datasets.py

```diff
@@ -10,7 +10,10 @@
 import time
 import numpy as np
 import deepchem as dc
-from kaggle_features import merck_descriptors 
+try:
+  from kaggle_features import merck_descriptors 
+except ImportError:
+  from kaggle.kaggle_features import merck_descriptors
```

Comment:
Ah, I see the issue here. This might work 

```
import sys
sys.path.append(".")
from kaggle_features import merck_descriptors
```

I think this should let you just use
```
from kaggle_features import merck_descriptors.
```